src/templates/IntegrationTemplate.js

```diff
@@ -0,0 +1,40 @@
+import React from "react"
```

Comment:
I meant to delete this file, but I am going to preserve it since it is in here now and don't want to delete anything you guys are reviewing. This was my original take at GraphQL and the thought process when I first started my research on it but reading more about it I realized this was more for blog post/list within Gatsby and opted to do the way that you see further up the page.